tests/contenttypes_tests/models.py

```diff
@@ -55,6 +55,21 @@ class Meta:
         proxy = True
 
 
+class GenericConcreteModel(models.Model):
```

Comment:
Is it possible to reuse existing models? There are also tests in `generic_relations` / `generic_relations_regress` that might have appropriate models.